### PR TITLE
fixes #12: It will be better to restrict user to add either "user ID" or "name", without leaving both details empty

### LINKED ISSUE #12: It will be better to restrict user to add either "user ID" or "name", without leaving both details empty
**Describe the bug**
At present, user is allowed to submit details without filling both user Id and name.It will be better to restrict user to add either "user ID" or "name", without leaving both empty.

**Screenshots**

ezgif com-video-to-gif(14)

**Would you like to work on the issue?**

- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by mustaqmustu**
Fixed #12 

Changes: changed insertData.java file.

Screenshots of the change: 
image

image

### CODE REVIEW

**Comment by mustaqmustu** [comment]
> Please use `setError` or something similar to give the alert when the user presses the button to proceed. Do not show a simple toast. The error should be shown on the EditText itself.

Done :)

**Comment by yashk2000** [comment]
@mustaqmustu thanks.

File: TempleApp/src/main/java/example/TempleApp/Activities/InsertData.java [hunk 1]
```diff
@@ -93,18 +93,23 @@ public void onClick(View v) {
         insert.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                id = id + uid1ET.getText().toString();
+                id = uid1ET.getText().toString();
```
Comment (Chromicle) [inline TempleApp/src/main/java/example/TempleApp/Activities/InsertData.java]: Please revert the change here as by adding `id` to we will know the type whether it is `Donation` or `Registration`  then check if the length equals to 3

Comment (yashk2000) [reply TempleApp/src/main/java/example/TempleApp/Activities/InsertData.java]: @mustaqmustu I did not notice that change. Please revert that in another pr ASAP.

Comment (Chromicle) [reply TempleApp/src/main/java/example/TempleApp/Activities/InsertData.java]: I reverted the PR please make another PR

**Comment by vchrombie** [comment]
You can simply revert the PR by using the revert button (below this [comment](https://github.com/amfoss/TempleApp/pull/23#issuecomment-500505881)), instead of opening a new PR.

**Pull request merged**